lib/process.js

```diff
@@ -27,19 +27,19 @@ function processScript (scriptPart, vueJestConfig, filePath) {
     return compileCoffeeScript(scriptPart.content, vueJestConfig, filePath)
   }
 
-  return compileBabel(scriptPart.content, undefined, undefined, vueJestConfig, filePath)
+  return compileBabel(scriptPart.content, undefined, undefined, vueJestConfig, filePath, babelTestObj)
 }
 
-module.exports = function (src, filePath, jestConfig) {
+// fourth argument is istanbul, it's injecting automatically
+module.exports = function (src, filePath, jestConfig, _, babelTestObj) {
   const vueJestConfig = getVueJestConfig(jestConfig)
 
   var parts = vueCompiler.parseComponent(src, { pad: true })
 
   if (parts.script && parts.script.src) {
     parts.script.content = fs.readFileSync(join(filePath, '..', parts.script.src), 'utf8')
   }
-
-  const result = processScript(parts.script, vueJestConfig, filePath)
+  const result = processScript(parts.script, vueJestConfig, filePath, babelTestObj)
```

Comment:
Can you please remove the `babelTestObj` from the code

Reply:
done